apps/fluent-tester/src/RNTester/TestComponents/Button/ButtonIconTest.tsx

```diff
@@ -0,0 +1,15 @@
+import { Button } from '@fluentui-react-native/button';
+import { Stack } from '@fluentui-react-native/stack';
+import * as React from 'react';
+import { View } from 'react-native';
+import { stackStyle } from '../Common/styles';
+
+export const ButtonIconTest: React.FunctionComponent<{}> = () => {
+  return (
+    <View>
+      <Stack style={stackStyle}>
+        <Button icon={require('./icon_24x24.png')} content="Button with Icon" tooltip="button tooltip" />
```

Comment:
Is this still a relevant icon for Fluent? I thought we moved to that one icon that looks like a wrench?

Reply:
I just needed *an* image for the test and didn't see any existing icon that looked appropriate to reuse.